core/src/com/fallenflame/game/enemies/EnemyModel.java

```diff
@@ -18,31 +25,77 @@
     public static final int CONTROL_MOVE_UP    = 0x04;
     /** Move the ship to the down */
     public static final int CONTROL_MOVE_DOWN  = 0x08;
-    /** Fire the ship weapon */
-    public static final int CONTROL_FIRE 	   = 0x10;
+    /** Move the ship to the down and left */
+    public static final int CONTROL_MOVE_DOWN_LEFT = 0x10;
+    /** Move the ship to the down and right */
+    public static final int CONTROL_MOVE_DOWN_RIGHT = 0x20;
+    /** Move the ship to the up and left */
+    public static final int CONTROL_MOVE_UP_LEFT = 0x40;
+    /** Move the ship to the up and right */
+    public static final int CONTROL_MOVE_UP_RIGHT = 0x80;
```

Comment:
Couple questions here: Why have you chosen the control codes you did? 
And can you add back control_fire? I had that there because I am working on adding an enemy that shoots. (And we want the control_fire hex code to be able to combine with any movement code in case we have an enemy that moves and shoots.

Also can you change all the "ship" to "enemy" (I know that's layover from when I added it, but since you're changing this anyways

Reply:
Control codes must be double of each other as it essentially does match statements by doing binary and operator. These are essentially the powers of 2 as hex codes.